gluten-arrow/src/main/java/org/apache/gluten/vectorized/NativePlanEvaluator.java

```diff
@@ -87,7 +86,7 @@ public ColumnarBatchOutIterator createKernelWithBatchIterator(
             new Spiller() {
               @Override
               public long spill(MemoryTarget self, Spiller.Phase phase, long size) {
-                if (!Spillers.PHASE_SET_SPILL_ONLY.contains(phase)) {
+                if (!Spiller.Phase.SPILL.equals(phase)) {
                   return 0L;
```

Comment:
ditto

Reply:
Java style recommends `if` if there is only one branch.

Reply:
Oh, these two are java classes.